test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -47,13 +47,13 @@ var (
 
 	tests = []struct{ nodes, existingPods, minPods int }{
 		{nodes: 500, existingPods: 500, minPods: 1000},
-		{nodes: 5000, existingPods: 5000, minPods: 1000},
+		{nodes: 5000, existingPods: 1000, minPods: 15000},
```

Comment:
Since we profile the whole test, we need to increase the number of pods being scheduled to get a representative profile.